### ISSUE TITLE
Cannot include helpers in liquid library

### ISSUE THREAD

**Issue by marol-hadidjaja** [padrino/padrino-framework#1907]
I have submodule named NewModule::App and it has helpers, one of them is pages_helper.rb. This helper have a method called **test_method**.
My Liquid Drop want to call this **test_method**, so I include ::NewModule::App::PagesHelper in this Liquid Drop but I got this error 
```
11:56:56 web.1    |   ERROR -  ArgumentError - wrong number of arguments (0 for 1..2):
11:56:56 web.1    |  /home/olivia/.rvm/gems/ruby-2.1.1@padrino/gems/padrino-core-0.12.2/lib/padrino-core/application/routing.rb:523:in `parent'
11:56:56 web.1    |   ERROR -  ArgumentError - wrong number of arguments (0 for 1..2):
11:56:56 web.1    |  /home/olivia/.rvm/gems/ruby-2.1.1@padrino/gems/padrino-core-0.12.2/lib/padrino-core/application/routing.rb:523:in `parent'
11:56:56 web.1    | /home/olivia/.rvm/gems/ruby-2.1.1@padrino/gems/padrino-core-0.12.2/lib/padrino-core/application/routing.rb:523:in `parent': wrong number of arguments (0 for 1..2) (ArgumentError)
11:56:56 web.1    | 	from /home/olivia/.rvm/gems/ruby-2.1.1@padrino/gems/activesupport-4.2.1/lib/active_support/dependencies.rb:500:in `load_missing_constant'
11:56:56 web.1    | 	from /home/olivia/.rvm/gems/ruby-2.1.1@padrino/gems/activesupport-4.2.1/lib/active_support/dependencies.rb:184:in `const_missing'
```

Just found that padrino-core gem has a problem in **parent** method. Can anyone help me to find the solution?

**Comment by ujifgc** [comment]
Looks like #1814

**Comment by marol-hadidjaja** [comment]
But it still not merged isn't it?

**Comment by namusyaka** [comment]
@marol-hadidjaja Thanks for the report, and sorry for the late reply.
[The patch](https://github.com/padrino/padrino-framework/commit/ebb8d22ccf7fa0d4cf8bba386f99014c5db4df4c) hasn't been merged into 0.12.x branch.
Could you try 0.13.0.beta1?
Please let us know if you are still seeing issues.

**Comment by nesquena** [comment]
I've just released 0.13.0.beta2 as well which also contains the fix.
